### DISCUSSION TITLE
您好 请问这是什么报错？ 我查阅了相关文档 并没有找到问题所在

### DISCUSSION THREAD

**Discussion by LmbADC** [xfgryujk/blivechat#236] [提问]
![Uploading 0194b606686e16276ca4fbdaae294539.png…]()

**Comment by xfgryujk** [comment]
图片没上传完啊

**Reply by LmbADC** [reply]
![0194b606686e16276ca4fbdaae294539](https://github.com/user-attachments/assets/922406ae-5de9-4092-b0d9-cae66927eaed)

**Reply by LmbADC** [reply]
出现很多次了。一直不懂是什么问题诶

**Reply by LmbADC** [reply]
一般就是突然断了 用了几分钟他就这样 然后OBS里面就不显示东西了

**Reply by LmbADC** [reply]
2024-09-23 11:15:09,565 WARNING [blivedm]: room=27689493 unknown cmd=LIVE_OPEN_PLATFORM_INTERACTION_END, command={'cmd': 'LIVE_OPEN_PLATFORM_INTERACTION_END', 'data': {'game_id': 'f6fab023-bbb9-4bc0-b0b1-0c74415372ed', 'timestamp': 1727061310}}

**Reply by luyanci** [reply]
> 2024-09-23 11:15:09,565 WARNING [blivedm]: room=27689493 unknown cmd=LIVE_OPEN_PLATFORM_INTERACTION_END, command={'cmd': 'LIVE_OPEN_PLATFORM_INTERACTION_END', 'data': {'game_id': 'f6fab023-bbb9-4bc0-b0b1-0c74415372ed', 'timestamp': 1727061310}}

`LIVE_OPEN_PLATFORM_INTERACTION_END`事件

在文档最后面
https://open-live.bilibili.com/document/f9ce25be-312e-1f4a-85fd-fef21f1637f8

**Reply by LmbADC** [reply]
怎么说 ，老大  @xfgryujk

**Reply by LmbADC** [reply]
这个有方法解决嘛请问

**Comment by xfgryujk** [comment]
过一段时间会自动重连吗

**Comment by xfgryujk** [comment]
看起来不是两种常见原因，如果能自动重连就不管了

**Reply by LmbADC** [reply]
不能

**Reply by LmbADC** [reply]
人呢哥

**Reply by xfgryujk** [reply]
> 人呢哥

这几天B站放假了不给查，你再发个断连的日志吧

**Comment by LmbADC** [comment]
今天一开就复现了老大 好像不一样了

**Comment by LmbADC** [comment]
![69431b1407a5e364db141c581ec9eabb](https://github.com/user-attachments/assets/3abf1a15-09d4-438c-a8e3-c11d7f23d2a6)

**Comment by LmbADC** [comment]
[blivechat11.1.log](https://github.com/user-attachments/files/17603431/blivechat11.1.log)

**Reply by xfgryujk** [reply]
这不是你自己关掉了房间页面吗

**Reply by LmbADC** [reply]
弹出来的那个浏览器页面不能关嘛？我之前关了也可以用呀

**Reply by LmbADC** [reply]
好我试一下
